gpu/people/tools/people_tracking.cpp

```diff
@@ -72,12 +73,27 @@ class PeopleTrackingApp
       interface->stop ();
     }
 
-    pcl::visualization::CloudViewer         viewer;
-    pcl::people::trees::MultiTreeLiveProc*  m_proc;
-    cv::Mat                                 m_lmap;
-    cv::Mat                                 m_cmap;
-    cv::Mat                                 cmap;
-    cv::Mat                                 m_bmap;
+    void load_tree(std::string treeFilenames[4], int numTrees)
+    {
+      std::ifstream fin0 (treeFilenames[0]);
+      assert(fin0.is_open());
+      m_proc = pcl::make_shared<pcl::people::trees::MultiTreeLiveProc>(fin0);
```

Comment:
```suggestion
      m_proc = std::make_unique<pcl::people::trees::MultiTreeLiveProc> (fin0);
```

MultiTreeLiveProc is a GPU class without eigen objects inside and pcl does not provide a `make_unique` method. Even if it did, it would spew a boost pointer which we do not need for this particular case.